apps/eec/sections/summary-data-sections.js

```diff
@@ -78,6 +78,24 @@ module.exports = {
         }
       }
     ]
-
+  },
```

Comment:
`/someone-else` step with`completing-for-someone-else` field is missing

Reply:
Added